### PR TITLE
feat: add custom theme & color-mode

### PR DESCRIPTION
**Pull request by CoderMoyv**
1. 添加自定义主题.
2. 添加@nuxtjs/color-mode切换颜色模式.
3. 添加切换颜色模式实例.

### CODE REVIEW

File: components/Examples.vue [hunk 1]
```diff
@@ -56,4 +60,10 @@ import zhCn from "element-plus/es/locale/lang/zh-cn";
 const timeValue = ref("");
 const hello = () => ElMessage.info("hello world");
 const helloSuccess = () => ElMessage.success("hello world");
+
+const color = useColorMode();
+const colorMode = computed({
```
Comment (YunYouJun) [inline components/Examples.vue:65]: Maybe we can refer it.

https://github.com/antfu/vitesse-nuxt3/blob/main/components/DarkToggle.vue

Comment (CoderMoyv) [reply components/Examples.vue:65]: This example uses button+function , But i use switch+computed. Do you mean to change to button+function?

File: components/Examples.vue [hunk 2]
```diff
@@ -56,4 +60,10 @@ import zhCn from "element-plus/es/locale/lang/zh-cn";
...
+  get: () => color.value === 'dark',
+    set: () => (color.preference = color.value === 'dark' ? 'light' : 'dark'),
```
Comment (YunYouJun) [inline components/Examples.vue:67]: Lint this indent.

File: components/Examples.vue (hunk 1 above)
Comment (YunYouJun) [reply components/Examples.vue:65]: Using switch is ok.

Comment (YunYouJun) [reply components/Examples.vue:65]: Using switch is ok.

File: components/Examples.vue [hunk 3]
```diff
@@ -1,4 +1,8 @@
 <template>
+  <el-switch v-model="colorMode" inline-prompt active-text="暗夜" inactive-text="白天" size="large"></el-switch>
```
Comment (YunYouJun) [inline components/Examples.vue:2]: I think we can use english keyword as text.

**Review by YunYouJun — CHANGES_REQUESTED** [review]
Modify it a little bit, and let's merge it.

**Review by YunYouJun — APPROVED** [review]

**Pull request merged**